misc/python/materialize/cli/ci_annotate_errors.py

```diff
@@ -750,7 +750,6 @@ def get_failures_on_main(test_analytics: TestAnalyticsDb) -> BuildHistory:
     try:
         build_history = test_analytics.build_history.get_recent_build_job_failures(
             pipeline=pipeline_slug,
-            branch="main",
```

Comment:
We might want to capture this information then in the function name: `get_recent_build_job_failures_on_main()`